seeders/usersSeeder.ts

```diff
@@ -76,10 +76,10 @@ const seedUsers = async (prisma: PrismaClient): Promise<void> => {
       document_number: random(100000, 999999999999).toString(),
       date_of_birth: faker.date.birthdate({ min: 18, max: 80, mode: "age" }),
       city: faker.address.cityName(),
-      contact_email: random(0, 1) === 1,
-      contact_sms: random(0, 1) === 1,
-      contact_wpp: random(0, 1) === 1,
-      user_rol: userRoles.appAdmin,
+      contact_email: random(0, 1) === 1 ? true : false,
+      contact_sms: random(0, 1) === 1 ? true : false,
+      contact_wpp: random(0, 1) === 1 ? true : false,
```

Comment:
But why using ternary operator? The strong comparison would do the job returning the boolean value of the operation.  As I mentioned above, consider `random(0,1) === 1`.